### ISSUE TITLE
Retire superseded LAS PDFs?

### ISSUE THREAD

**Issue by esilvia** [ASPRSorg/LAS#88]
Google and Bing web searches for "LAS 1.4 specification" both link directly to old PDF versions of the LAS 1.4 specification.

## Bing
![image](https://user-images.githubusercontent.com/11743012/69199350-8c434580-0aec-11ea-89a6-c8e11229a0b6.png)

## Google
![image](https://user-images.githubusercontent.com/11743012/69199373-a1b86f80-0aec-11ea-80f6-515dd57d4ce9.png)

The most recent version (R15) doesn't even show up on web searches. This could potentially result in users inadvertently relying on outdated versions of the LAS specification. 

I'm able to replace the existing link to something like "LAS_1_4_latest.pdf" so that the URL to the current version is static between versions. In addition, I'd like to remove the superseded versions of the LAS specification from the ASPRS website and instead archive the PDFs on the GitHub wiki, leaving only the latest version on the ASPRS website.

Is there any reason not to remove the old links? Will this negatively impact anyone?

**Comment by esilvia** [comment]
Here's where those links are found: https://www.asprs.org/divisions-committees/lidar-division/laser-las-file-format-exchange-activities

**Comment by rapidlasso** [comment]
Tough call. I get the pragmatism, but to simply erase history because of some search engine's ranking algorithm seems the wrong solution. Maybe we could leave them up but reorganize them to where they are stored so that older revisions are less likely to show up as the first search result in the future?

**Comment by kdamkjer** [comment]
I agree, Martin. Shouldn't a robots.txt keep SEs from indexing (for well behaved SEs)? Apply to an "obsoleted" directory, perhaps? That way we can keep older revisions directly linked. Thoughts?

Sent from my Android phone using TouchDown (www.symantec.com)

-----Original Message-----

**Comment by esilvia** [comment]
@kdamkjer You're right that a well behaved robots.txt file should be able to hide the old revisions, but I don't know enough personally to pull that off. I'll ask around at ASPRS and see if there's anyone with those skills.
@rapidlasso That's pretty much what I was thinking. We could keep only the latest revision on the ASPRS website, while archiving every revision's PDF on the GitHub Release or Wiki page for historical purposes. I have no intention to remove history.

**Comment by esilvia** [comment]
From today's conference call:

I'll proceed with having a LAS_1_4_latest.pdf available from the ASPRS website and archive the previous revisions of LAS 1.4 on the GitHub page in some capacity. I'll try to do this before the end of the month so that you folks can comment.
